### ISSUE TITLE
Please update server install script...

### ISSUE THREAD

**Issue by Draylynn** [Chuunkyz/BenchmarkII-1.18#129]
The install for the latest Benchmark II on CurseForge's website, for the server, is out of date. The API it is using is out of date and shut down.
Please migrate to the new API: https://docs.curseforge.com/

**Comment by deathbows2me** [comment]
Yeah, just confirmed this is my issue. I am installing a new server and the API does not work to install the mods.
